docs/users/cli-usage.md

```diff
@@ -0,0 +1,146 @@
+<h1>Using the ddev command line interface (CLI)</h1>
+
+Type `ddev` in a terminal to see the available ddev commands:
+
+```
+➜  ddev
+This Command Line Interface (CLI) gives you the ability to interact with the ddev to create a local development environment.
+
```

Comment:
At some point I'd love to see this generated from the ddev command and use an include to bring it in. It seems readthedocs does support that, but perhaps not with our markdown approach? 

Note that we can update the circle build to *push* to readthedocs if we need to, and it could generate output like this. I just hate having stuff like this buried in here that is guaranteed to go stale regularly. 

No action required here, just speculating and proposing.